hc/front/tests/test_add_check.py

```diff
@@ -12,3 +12,18 @@ def test_it_works(self):
         assert Check.objects.count() == 1
 
     ### Test that team access works
+    def test_team_access(self):
+        self.check = Check(user=self.alice)
+        self.check.save()
+
+        self.client.login(username="bob@example.org", password="password")
+
+        url = "/checks/%s/timeout/" % self.check.code
+        payload = {"timeout": 5000, "grace": 60}
+
+        r = self.client.post(url, data=payload)
```

Comment:
Refrain from using variable names such as `r`. Use a descriptive name. If you find such in existing code, try to understand the context and update them to a meaningful name.

@georgreen @davidmukiibi @victoriaaoka @Andretalik @Kachulio1